server/src/main/java/org/elasticsearch/TransportVersion.java

```diff
@@ -123,22 +123,10 @@ private static TransportVersion registerTransportVersion(int id, String uniqueId
     public static final TransportVersion V_8_8_0 = registerTransportVersion(8_08_00_99, "f64fe576-0767-4ec3-984e-3e30b33b6c46");
     public static final TransportVersion V_8_8_1 = registerTransportVersion(8_08_01_99, "291c71bb-5b0a-4b7e-a407-6e53bc128d0f");
 
-    public static final TransportVersion V_8_9_0 = registerTransportVersion(8_09_00_99, "13c1c2cb-d975-461f-ab98-309ebc1c01bc");
```

Comment:
How  can we remove this released ES version? Even if not used, should we not keep it?

Reply:
Ah ok, that is indeed misleading. For convenience, should we not have a constant that encodes this? Or maybe that is just not useful at this point? e.g.

```
public static final TransportVersion V_8_9_0 = V_8_500_020;
```

Reply:
There's a comment above the TV that was used whan a release was branched